cv32e40p/tests/programs/custom/custom_opcode_illegal_test/custom_opcode_illegal_test.S

```diff
@@ -0,0 +1,5151 @@
+.include "user_define.h"
```

Comment:
Please add an SPDX license header.

Reply:
@XavierAubert , please help to change as per request.

Reply:
done  98ba6361b664c867784fd77fa7fd831cbedc2022